src/components/pages/dashboard-pages/HomeStudent.jsx

```diff
@@ -1,15 +1,37 @@
 import { Link } from 'react-router-dom';
```

Comment:
On line 68. Would it be better to conditionally render "Parents info" instead of using a ternary?

Reply:
Sorry, this file is not from this pr. I pulled all changes into my branch to avoid conflicts later. 
I believe all these changes related to the dashboard were already approved PP-35 dashboard https://github.com/Code-the-Dream-School/gg-pac-team4-front/pull/13